resources/jxbrowser/jxbrowser.properties.template

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright 2020 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# Replace <KEY> with JxBrowser license key and save this file as jxbrowser.properties
+jxbrowser.license.key=<KEY>
```

Comment:
You should either add a newline here or specify in the instructions that no newline should be added. Everyone's probably going to add one unless told not to.